RickLeinecker/Fortuna
# Battleground intro animation

**Description**
Made a hot intro cutscene that looks like this:
https://drive.google.com/open?id=1hwgG-Cu8wqZLT7bMG0iDcuF6wXr7s3-_


Resolves #199 

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
Make a bulleted/numbered list of steps to verify this feature/bugfix. Include screenshots if necessary.
Passes flow. Works as indicated on animated gif.

**Final Checklist:**
Go down the list and check them off.
